eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -106,6 +106,10 @@ jobs:
       # NET472
       - ${{ if eq(parameters.jobParameters.framework, 'net472') }}:
         - Windows.10.Amd64.Client19H1.Open
+
+      # SourceBuild
+      - ${{ if eq(parameters.jobParameters.framework, 'sourceBuild') }}:
+        - RedHat.7.Amd64.Open
```

Comment:
This should not be needed, these are the helix queues that we use for running tests, we don't run tests in sourceBuild. We shouldn't need to add any reference to any queue as that is handled by `platform-matrix`.